cloudplatform/cloudplatform-connectivity/src/main/java/com/sap/cloud/sdk/cloudplatform/connectivity/DefaultHttpDestination.java

```
@@ -949,9 +949,9 @@ public DefaultHttpDestination build()
 
             // handle proxy type == OnPremise
             if( builder.get(DestinationProperty.PROXY_TYPE).contains(ProxyType.ON_PREMISE) ) {
-                final DefaultHttpDestination proxyDestination = proxyHandler.handle(this);
-                if( proxyDestination != null ) {
-                    return proxyDestination;
+                final Try<DefaultHttpDestination> proxyDestination = Try.of(() -> proxyHandler.handle(this));
+                if( proxyDestination.isSuccess() ) {
+                    return proxyDestination.get();
```

It feels pretty wrong to silently ignore any exception that might occur here... customers can't even refer to the application logs to see what might have gone wrong.

---

I added a log here.